src/shared/SRP6/SRP6.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of the CMaNGOS Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+/** \file
+    \ingroup realmd
+*/
+
+#include "Common.h"
+#include "Auth/HMACSHA1.h"
+#include "Auth/base32.h"
+#include "SRP6.h"
+#include "Log.h"
+
+#include <openssl/md5.h>
+#include <ctime>
+#include <utility>
+
```

Comment:
This list of includes is probably a bit overkill. What do we need Log.h for, or md5.h or ctime?